docs/guides/lint-arazzo.md

```diff
@@ -0,0 +1,73 @@
+# Lint Arazzo with Redocly CLI
+
+Arazzo is an open standard for describing multi-step API calls in sequence.
+While under development, this standard was also known as "OpenAPI Workflows".
+Redocly CLI offers support for checking that your Arazzo files are valid.
+
+{% admonition type="info" name="Experimental Arazzo support" %}
+This feature is at an early stage, please send us lots of feedback!
+{% /admonition %}
+
+## Lint an Arazzo file
+
+Use your existing Arazzo files, or use the examples in the [Museum API project](https://github.com/Redocly/museum-openapi-example) if you'd prefer to use sample data to try things out.
```

Comment:
Suggestion: "...use the examples..." --> "...use the Arazzo examples..."

On first read, I thought it was just recommending readers go there and use the data to build their own, but we already have completed examples (...awesome!)

Reply:
Added (All the credit for examples goes to @DmitryAnansky )